documentation/docs/reference/services/Profile.md

```diff
@@ -19,20 +17,15 @@ Response format:
     "firstName": "John",
     "lastName": "Doe",
     "points": 2021,
-    "timezone": "Americas UTC+8",
-    "avatarUrl": "https://github.com/.../profile.jpg",
-    "discord": "patrick#1234",
-    "teamStatus": "LOOKING_FOR_TEAM",
-    "description": "Lorem Ipsum…",
-    "interests": ["C++", "Machine Learning"]
+    "isVirtual": false
```

Comment:
We need to add back the avatar URL, so we can randomly generate a profile picture and affix it to the profile.

Reply:
> we can randomly generate a profile picture and affix it to the profile

So, are we setting this to some URL on profile creation?